.github/workflows/build.yml

```diff
@@ -13,20 +13,20 @@ env:
 jobs:
   push-ghcr:
     name: Build and push image
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
```

Comment:
change this to 24.04 to fix the tar bug.

Reply:
thanks